### PR TITLE
Rename FirmwareUpdater to FirmwareUploader

### PR DESCRIPTION
**Pull request by umbynos**
Updated binary name (from FirmwareUpdater to FirmwareUploader) in the CI / Taskfiles / .gitignore.
I think it's a bit confusing to use both FirmwareUploader and FirmwareUpdater in the same GitHub repo though.. I've renamed also the repo handle. The only thing missing is to rename the repo name in settings.

### CODE REVIEW

**Comment by umbynos** [comment]
CI checks are failing because I changed also links to the repo.

File: README.md [hunk 1]
```diff
@@ -6,63 +6,67 @@ Use this tool to update the firmware and/or add SSL certificates for any WINC, N
...
-https://github.com/arduino/FirmwareUpdater/releases/latest
+https://github.com/arduino/FirmwareUploader/releases/latest
 
 ## Usage
 
 Extract the zip file and run (for example, NINA -> WiFi1010)
 
 ```
-./$your_os/updater -flasher firmwares/NINA/FirmwareUpdater.mkrwifi1010.ino.bin -firmware firmwares/NINA/1.2.1/NINA_W102.bin -port /dev/ttyACM0  -address arduino.cc:443 -restore_binary /tmp/arduino_build_619137/WiFiSSLClient.ino.bin -programmer {runtime.tools.bossac}/bossac
+./FirmwareUploader -flasher firmwares/NINA/FirmwareUpdater.mkrwifi1010.ino.bin -firmware firmwares/NINA/1.2.1/NINA_W102.bin -port /dev/ttyACM0 -address arduino.cc:443 -restore_binary /tmp/arduino_build_619137/WiFiSSLClient.ino.bin -programmer {runtime.tools.bossac}/bossac
```
Comment (silvanocerza) [inline README.md:16]:
```suggestion
./FirmwareUploader -flasher firmwares/NINA/FirmwareUploader.mkrwifi1010.ino.bin -firmware firmwares/NINA/1.2.1/NINA_W102.bin -port /dev/ttyACM0 -address arduino.cc:443 -restore_binary /tmp/arduino_build_619137/WiFiSSLClient.ino.bin -programmer {runtime.tools.bossac}/bossac
```

File: README.md [hunk 2]
```diff
@@ -6,63 +6,67 @@ Use this tool to update the firmware and/or add SSL certificates for any WINC, N
...
 ```
 
 To flash a MKR1000:
 
 ```
-./$your_os/updater -flasher firmwares/WINC1500/FirmwareUpdater.mkr1000.ino.bin -firmware firmwares/WINC1500/19.5.4/m2m_aio_3a0.bin -port /dev/ttyACM0  -address arduino.cc:443 -restore_binary /tmp/arduino_build_619137/WiFiSSLClient.ino.bin -programmer {runtime.tools.bossac}/bossac
+./FirmwareUploader -flasher firmwares/WINC1500/FirmwareUpdater.mkr1000.ino.bin -firmware firmwares/WINC1500/19.5.4/m2m_aio_3a0.bin -port /dev/ttyACM0 -address arduino.cc:443 -restore_binary /tmp/arduino_build_619137/WiFiSSLClient.ino.bin -programmer {runtime.tools.bossac}/bossac
```
Comment (silvanocerza) [inline README.md:22]:
```suggestion
./FirmwareUploader -flasher firmwares/WINC1500/FirmwareUploader.mkr1000.ino.bin -firmware firmwares/WINC1500/19.5.4/m2m_aio_3a0.bin -port /dev/ttyACM0 -address arduino.cc:443 -restore_binary /tmp/arduino_build_619137/WiFiSSLClient.ino.bin -programmer {runtime.tools.bossac}/bossac
```

**Review by silvanocerza — CHANGES_REQUESTED** [review]
Just two small changes to do.

Also shouldn't we change `extras/package_index.json.template` too? The tool there is still called `fwupdater`.

**Comment by umbynos** [comment]
I did not rename these two files and not updated the readme accordingly because they are used [in the js lib](https://github.com/arduino/arduino-create-agent-js-client/blob/95383da47c5b02dd29f78410c38cd909a05e0b8a/src/firmware-updater.js).

Regarding the template:
the name fwupdater will be changed in the future but it will eventually cause breaking changes.

**Review by silvanocerza — APPROVED** [review]

**Pull request merged**